tableland-openapi-spec.yaml

```diff
@@ -46,6 +46,14 @@ paths:
                   params:
                     - id: "0"
                       create_statement: "CREATE TABLE myname_31337 (column_a int, column_b text);"
+              validateWriteQuery:
```

Comment:
The Local Tableland tests that are auto generated from this file all pass with these addtions, but doesn't hurt to double check if anything looks incorrect